### ISSUE TITLE
Duplicated entry on Server Pro Tips PDF file

### ISSUE THREAD

**Issue by DougSmythies** [canonical/ubuntu.com#12053]
There was [a launchpad bug report](https://bugs.launchpad.net/serverguide/+bug/1990768) filed under the Ubuntu Server Guide that should have been reported here.

---
*Reported from: https://ubuntu.com/download/server

**Comment by mtruj013** [comment]
Thanks for reporting this @DougSmythies !

To summarize [this PDF](https://launchpadlibrarian.net/625624926/Ubuntu.Server.CLI.pro.tips.19.04.22.png) lists `Remove the package` `sudo apt remove <package>` twice. OP has suggested that on of them could be removed or updated to the correct command.

**Comment by petesfrench** [comment]
@bogdans-canonical Is updating a pdf a brand issue?

**Comment by bogdans-canonical** [comment]
Yes, @petesfrench , but it should be submitted as a brief to canonical-editors first. Thanks,

**Comment by muhammad-ali-pk** [comment]
@DougSmythies Closing this issue because the latest cheat sheet available on the download page has this mistake corrected.
Please feel free to reopen the issue if otherwise.
